pkgs/top-level/all-packages.nix

```diff
@@ -9917,6 +9917,11 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) IOKit;
   };
 
+  pcscliteWithPolkit = pcsclite.override {
```

Comment:
I'm not a fan of adding new top level names like this, but I don't know a simpler solution.
Ideally pcscd should detect/use polkit at runtime, but it doesn't, so we should provide both variants.